evalml/tests/automl_tests/test_default_algorithm.py

```diff
@@ -922,3 +927,52 @@ def test_default_algorithm_accepts_URL_email_features(
         assert pipeline.parameters["Categorical Pipeline - Select Columns Transformer"][
             "columns"
         ] == ["url", "email"]
+
+
+@pytest.mark.parametrize("input_type", ["pd", "ww"])
+@pytest.mark.parametrize("problem_type", ProblemTypes.all_problem_types)
+def test_exclude_featurizers_default_algorithm(
+    problem_type,
+    input_type,
+    get_test_data_from_configuration,
+):
+    parameters = {}
+    if is_time_series(problem_type):
+        parameters = {
+            "time_index": "dates",
+            "gap": 1,
+            "max_delay": 1,
+            "forecast_horizon": 3,
+        }
+
+    X, y = get_test_data_from_configuration(
+        input_type, problem_type, column_names=["dates", "text", "email", "url"]
+    )
+
+    algo = DefaultAlgorithm(
+        X,
+        y,
+        problem_type,
+        sampler_name=None,
+        search_parameters={"pipeline": parameters},
+        exclude_featurizers=[
+            "DatetimeFeaturizer",
+            "EmailFeaturizer",
+            "URLFeaturizer",
+            "NaturalLanguageFeaturizer",
+            "TimeSeriesFeaturizer",
+        ],
+    )
+
+    pipelines = []
+    for _ in range(4):
+        pipelines.extend([pl.name for pl in algo.next_batch()])
+
+    # A check to make sure we actually retrieve constructed pipelines from the algo.
+    assert len(pipelines) > 0
+
+    assert not any([DateTimeFeaturizer.name in pl for pl in pipelines])
```

Comment:
could you repeat the changes you've made to `test_exclude_featurizers` here and in `test_exclude_featurizers_iterative_algorithm` as well? Thanks!

Reply:
Updated here: [f115377](https://github.com/alteryx/evalml/pull/3631/commits/f115377ae5e23dbe4789bc33f5e03456c5ae6f27)